setlexsem/utils.py

```diff
@@ -79,7 +79,7 @@ def get_prompt_file_path(sampler_hp, prompt_hp, random_seed):
         random_seed,
     )
     experiment_folder_structure = os.path.join(
-        sampler_hp["set_type"],
+        sampler_hp["set_types"],
```

Comment:
Did the move from `set_type` to `set_types` result in config files that have `n` (which is just for sampling numbers) even when `set_types = ["words", "numbers"]`, even though `n` is not applicable for words?

Reply:
that was an issue, but we also were calling the key "set_type" at times, and sometimes "set_types". I refactors all of it to "set_types" now.